src/Cafe/HW/Latte/Renderer/Vulkan/TextureReadbackVk.cpp

```diff
@@ -79,6 +79,16 @@ uint32 LatteTextureReadbackInfoVk::GetImageSize(LatteTextureView* textureView)
 		// todo - if driver does not support VK_FORMAT_D24_UNORM_S8_UINT this is represented as VK_FORMAT_D32_SFLOAT_S8_UINT which is 8 bytes
 		return baseTexture->width * baseTexture->height * 4;
 	}
+	else if (textureView->format == Latte::E_GX2SURFFMT::R5_G6_B5_UNORM )
+	{
+		cemu_assert(textureFormat == VK_FORMAT_R8G8B8A8_UNORM);
+		return baseTexture->width * baseTexture->height * 4;
+	}
+	else if (textureView->format == Latte::E_GX2SURFFMT::R5_G5_B5_A1_UNORM )
```

Comment:
For `R5_G5_B5_A1_UNORM` conversion is always necessary since the backend doesn't use Vulkan formats that are bit-compatible. See code [here](https://github.com/cemu-project/Cemu/blob/90eb2e01f405e502ed33fea330352e167c4a14db/src/Cafe/HW/Latte/Renderer/Vulkan/VulkanRenderer.cpp#L2383). Maybe just drop this case for now?

Reply:
Removed